qubekit/molecules/ligand.py

```diff
@@ -1405,9 +1405,9 @@ def _build_offxml_angles(self, offxml: ForceField):
 
         angle_types = self.angle_types
         for angles in angle_types.values():
-            graph = ClusterGraph(
-                mols=[rdkit_mol],
-                smirks_atoms_lists=[angles],
+            graph = SingleGraph(
+                mol=rdkit_mol,
+                smirks_atoms=angles[0],
```

Comment:
So instead of all the angles grouped together, we create one smirk for the first angle of this angle type? I cannot quite see how the smirk is general enough to apply to all these angles grouped together.